lib/resources/Subscriptions.js

```diff
@@ -3,15 +3,37 @@
 var StripeResource = require('../StripeResource');
 var stripeMethod = StripeResource.method;
 
+function encode(data) {
+  if (Array.isArray(data.items)) {
+    var items = {};
+    data.items.map(function(item, i) {
```

Comment:
Can we put this in `utils` under `arrayToObject` a little like in Ruby? (and with a test)